homeassistant/components/accuweather/coordinator.py

```diff
@@ -75,7 +75,14 @@ async def _async_update_data(self) -> dict[str, Any]:
             async with timeout(10):
                 result = await self.accuweather.async_get_current_conditions()
         except EXCEPTIONS as error:
-            raise UpdateFailed(error) from error
+            raise UpdateFailed(
+                translation_domain=DOMAIN,
+                translation_key="update_error",
+                translation_placeholders={
+                    "data_type": "current conditions data",
```

Comment:
Can we be more specific? Because this would look really strange in translated languages:

"Mislukt om current conditions data op te halen"

Reply:
Good point!